includes/unsupported-theme-handlers/class-sensei-unsupported-theme-handler-page-imitator.php

```diff
@@ -53,7 +53,7 @@ protected function prepare_wp_query( $wp_query, $object_to_copy, $post_params )
 	 *                               creating the Page.
 	 */
 	protected function output_content_as_page( $content, $object_to_copy = null, $post_params = array() ) {
-		global $post, $wp_query;
+		global $post, $wp_query, $wp_the_query;
```

Comment:
No, but I thought I would sneak it in since I noticed it was missing. Otherwise, it is possible for it to cause issues in the future (since we use `$wp_the_query` later in this function).

Reply:
OK, may be best as a separate commit next time with its own explanation to avoid any possible confusion. 🙂